app/src/main/java/com/myapps/toualbiamine/basketbusiness/Model/User.java

```diff
@@ -0,0 +1,38 @@
+package com.myapps.toualbiamine.basketbusiness.Model;
+
+
+public class User {
+        private String name;
+        private String password;
+        private String email;
+        private boolean isStaff;
+
+    public User() {
+
+}
```

Comment:
For formatting, could you put this `}` on the same line as the opening `{`? 
The function is empty so we don't need to have an empty body with blank lines.